tests/quantizers_infrastructure_tests/pytorch_tests/base_pytorch_infrastructure_test.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import unittest
+
+from typing import Dict
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from model_compression_toolkit.core.common.quantization.node_quantization_config import NodeWeightsQuantizationConfig, \
+    NodeActivationQuantizationConfig
+from model_compression_toolkit import quantizers_infrastructure as qi, QuantizationConfig
+from model_compression_toolkit.core.common.target_platform import QuantizationMethod
+from model_compression_toolkit.core.tpc_models.default_tpc.latest import get_op_quantization_configs
+
+
+class ZeroWeightsQuantizer(qi.BasePytorchTrainableQuantizer):
+    """
+    Trainable constrained quantizer to quantize a layer inputs.
```

Comment:
This comment is copied automatically every time we implement a new quantizer and is not relevant here. 
Please edit it or remove it completely (it's a test so it's not mandatory here, but is this case maybe some explanation would be good).